org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/LaunchingPlugin.java

```diff
@@ -582,6 +584,11 @@ public void saving(ISaveContext context1) throws CoreException {
 		ResourcesPlugin.getWorkspace().addResourceChangeListener(this, IResourceChangeEvent.PRE_DELETE | IResourceChangeEvent.PRE_CLOSE);
 		DebugPlugin.getDefault().getLaunchManager().addLaunchListener(this);
 		DebugPlugin.getDefault().addDebugEventListener(this);
+		IEclipsePreferences instanceNode = InstanceScope.INSTANCE.getNode(getBundle().getSymbolicName());
+		IEclipsePreferences defaultNode = DefaultScope.INSTANCE.getNode(getBundle().getSymbolicName());
+		if (instanceNode.getBoolean(PREF_DETECT_VMS_AT_STARTUP, defaultNode.getBoolean(PREF_DETECT_VMS_AT_STARTUP, true))) {
+			new DetectVMInstallationsJob().schedule(200);
```

Comment:
Delay of 200 is needed because of ... ? Do we need it at all?

Reply:
Ack